Assets/Scripts/Player/PlayerMoney.cs

```diff
@@ -25,7 +25,7 @@ private void Awake()
     private void Start()
     {
         LoadFromFile();
-        moneyText.text = money.ToString();
+        //moneyText.text = money.ToString();
```

Comment:
Als je deze uitcomment dan werkt het geld laten zien niet meer.

Reply:
Working build. PlayerMoney is next up the list to be reworked.